app/controllers/answers_controller.rb

```diff
@@ -1,11 +1,37 @@
 class AnswersController < ApplicationController
 
-  def index
+  def new
+    @question = Question.find(params[:question_id])
+    @answer = Answer.new
   end
 
-  def show
+  def create
+    question = Question.find(params[:question_id])
+    answer = question.answers.new(content: params[:answer][:content], user_id: session[:user_id])
```

Comment:
Won't it save the content as the whole hash ?

2015-03-20 15:02 GMT-04:00 Steven G. Harms <notifications@github.com>:

> In app/controllers/answers_controller.rb
> <https://github.com/nyc-fiddler-crabs-2015/LambOverice/pull/3#discussion_r26867155>
> :
>
> >    end
> >
> > -  def show
> > +  def create
> > +    question = Question.find(params[:question_id])
> > +    answer = question.answers.new(content: params[:answer][:content], user_id: session[:user_id])
>
> seems like you should be able to say question.answers.new(params[:answer]).
> The double dereference [][] seems like a smell.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nyc-fiddler-crabs-2015/LambOverice/pull/3/files#r26867155>
> .
>